src/class/prte_hash_table.c

```diff
@@ -87,6 +88,8 @@
  *
  */
 
+#define HASH_SHOW_REPLACEMENT_WARNING 0
```

Comment:
I guess the idea is that if somebody really wants to see the overwrite warnings they flip the define. Maybe a comment to that effect?

Reply:
I'll add that.
I originally just had the warnings, but they were popping up quite a bit. I am going back through now to see if I can eliminate the duplicates at the higher level. Then maybe we can just leave this enabled by default.

Reply:
Done.